packages/platform-browser/src/browser/meta.ts

```diff
@@ -63,17 +93,37 @@ export class Meta {
     }, []);
   }
 
+  /**
+   * Retrieves a `<meta>` tag element in the current HTML document.
+   * @param attrSelector The tag attribute and value to match against, in the format
+   * `"tag_attribute=`value string`"`.
```

Comment:
```suggestion
   * `"tag_attribute='value string'"`.
```
I don't think that the nested backticks is right (here and a number of other places), unless you are trying to do something with ES6 string templates?

Reply:
Oh I see in the preview, the backticks are removed, but I still think that it is confusing.

Perhaps it would be clearer to just explain how these selectors are used with [`querySelector`](https://developer.mozilla.org/docs/Web/API/Document/querySelector) in the following format: `meta[${attrSelector}]`. It may be worth adding a "See Also" link for this (https://developer.mozilla.org/docs/Web/API/Document/querySelector) as well?

Perhaps we can just explain the `attrSelector` formatting a single time in the class documentation and then refer to it from the various functions that use `attrSelector`?